src/svd.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+from data2img import save_single_img, get_zoomed_image
+
+
+def decompose(image_matrix):
+    """
+    # Get decomposed matrices, USVtranspose
+    """
+    U, S, V = np.linalg.svd(image_matrix, full_matrices=True)
```

Comment:
Very little syntax thing, no need for full_matrices=True

def svd(a, full_matrices=1, compute_uv=1):
    """
    Singular Value Decomposition.

Default is full_matrices = True as in python:     1 == True :-)